app/services/open_library.rb

```diff
@@ -0,0 +1,37 @@
+class OpenLibrary
+  require 'faraday'
+  require 'json'
+
+  def initialize(isbn)
+    @isbn = isbn
+    @options = { bibkeys: "ISBN:#{isbn}", format: 'json', jscmd: 'data' }
+    @base_url = 'https://openlibrary.org/api'
```

Comment:
Use credentials to save the url https://blog.saeloun.com/2019/10/10/rails-6-adds-support-for-multi-environment-credentials.html#in-rails-6